src/epd1in54/graphics.rs

```diff
@@ -79,7 +90,12 @@ mod tests {
     fn graphics_rotation_0() {
         let mut display = Display1in54::default();
         let _ = Line::new(Point::new(0, 0), Point::new(7, 0))
-            .into_styled(PrimitiveStyle::with_stroke(Black, 1))
+            .into_styled(
+                PrimitiveStyleBuilder::new()
+                    .stroke_color(Black)
+                    .stroke_width(1)
+                    .build(),
+            )
```

Comment:
I think I would have stayed with the shorter variant here since it is still working. (At least if I understand this correctly: https://docs.rs/embedded-graphics/0.7.0/embedded_graphics/primitives/struct.PrimitiveStyle.html)